### ISSUE TITLE
Bug: no zsh_history in sub shell

### ISSUE THREAD

**Issue by codejnki** [sbstp/kubie#100]
When I start a new `kubie ctx` session it has no history.  This appears to be similar to #31.  

Looking a PR #32 and comparing it to the current state of https://github.com/EItanya/kubie/blob/master/src/shell/zsh.rs it appears that the fix may no longer be in the current release.

Can the commit from #32 get merged into `master` and a new release cut?

**Comment by ghostsquad** [comment]
yes please! I really need this. I've found myself _not_ wanting to be in a kubie shell because my history doesn't get saved. :(

**Comment by simonst** [comment]
It would be really helpful to have this in again!

**Comment by rvignesh89** [comment]
I had the same issue and it turns out history is indeed copied over to the sub-shells if the .zsh_history is correctly populated - https://github.com/sbstp/kubie/blob/master/src/shell/zsh.rs#L40-L43

It turns out you can set the location of the history file by setting the `HISTFILE` var (https://ryantoddgarza.com/a-better-zsh-history-pt1) Somehow this was configured incorrectly in my shell and overriding that to `~/.zsh_history` did the trick for me. Not sure if it's the same issue for others in this thread though.

**Comment by megumish** [comment]
thanks @rvignesh89 .  I set up the HISTFILE and it works!

However, I still think #32 should be merged into the release for those who want to run without this configuration.

**Comment by codejnki** [comment]
I was able to pull master and build it locally and now history is present.  So that could be an option for some people.

I do think we could benefit from a fresh release being cut and pushed out to the package managers.

**Comment by sbstp** [comment]
I'll create a new release

**Comment by CharlesB2** [comment]
This was driving me nuts until I found that it was kubie causing to lose history

**Comment by korncola** [comment]
it still an issue for me

**Comment by completenovice** [comment]
Hi, is there any update on this? It is still an issue.
